qcore/timeseries.py

```diff
@@ -897,3 +897,20 @@ def acc(self, station, comp=Ellipsis):
             np.frombuffer(o, dtype=np.int32).astype(np.float32).reshape(-1, 3)[:, comp]
             / self.stations.scale[i]
         )
+
+
+def get_observed_stations(observed_data_folder):
+    """
+    returns a list of station names that can be found in the observed data folder
+    
+    :param observed_data_folder: path to the record folder, e.g. observed/events/vol1/data/accBB/
+    :type observed_data_folder: str/os.path
+    :return: list of unique station names
+    :rtype: list[str]
+    """
+    search_path = os.path.abspath(os.path.join(observed_data_folder, "*"))
+    files = glob(search_path)
+    station_names = set(
+        [os.path.splitext(os.path.basename(filename))[0] for filename in files]
+    )
```

Comment:
Possible github glitch, so I'll put this in again:
This should use a set comprehension instead of converting a list comprehension